test/e2e/instance-create.e2e.ts

```diff
@@ -509,25 +509,25 @@ test('attaching additional disks allows for combobox filtering', async ({ page }
   await attachExistingDiskButton.click()
   await selectADisk.click()
   // several disks should be shown
-  await expect(page.getByRole('option', { name: 'disk-0001' })).toBeVisible()
-  await expect(page.getByRole('option', { name: 'disk-0002' })).toBeVisible()
-  await expect(page.getByRole('option', { name: 'disk-1000' })).toBeVisible()
+  await expect(page.getByRole('option', { name: 'disk-0005' })).toBeVisible()
```

Comment:
After seeding the disks with a mix of states, not all of the previous items were visible. Now the random numbers are deterministic, and so the same disks should be available each time – the only way I can see this breaking is if we fiddle with the `Rando` numbers.